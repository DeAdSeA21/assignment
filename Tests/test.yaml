dueDate: 2025-04-28
latePenalties: [10, 20, 100]
test:
  launch: ./loan
  timelimit: 10
  grading:
    - oracle: smart
      scoring: ByLine
      ws: true
      cap: 100
    - oracle: smart
      scoring: ByLine
      numbersOnly: true
      cap: 85
build:
  instructorFiles:
    exclude:
      - loan.cpp
  studentFiles:
    exclude:
      - makefile
  command: make -k
