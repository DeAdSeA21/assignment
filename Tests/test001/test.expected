Starting loan balance: Annual interest rate: Monthly payment: 
Month	Balance
1	9993.75
2	9987.47
3	9981.17
4	9974.84
5	9968.48
6	9962.09
7	9955.67
8	9949.23
9	9942.76
10	9936.26
11	9929.73
12	9923.17
13	9916.58
14	9909.97
15	9903.32
16	9896.65
17	9889.95
18	9883.22
19	9876.46
20	9869.67
21	9862.85
22	9856.00
23	9849.12
24	9842.21
25	9835.26
26	9828.29
27	9821.29
28	9814.26
29	9807.20
30	9800.11
31	9792.98
32	9785.82
33	9778.64
34	9771.42
35	9764.17
36	9756.89
37	9749.57
38	9742.23
39	9734.85
40	9727.44
41	9720.00
42	9712.52
43	9705.02
44	9697.47
45	9689.90
46	9682.29
47	9674.65
48	9666.98
49	9659.27
50	9651.53
51	9643.76
52	9635.95
53	9628.11
54	9620.23
55	9612.32
56	9604.37
57	9596.39
58	9588.38
59	9580.33
60	9572.24
